build-system/tasks/helpers.js

```diff
@@ -219,12 +219,16 @@ function appendToCompiledFile(srcFilename, destFilePath) {
   }
 }
 
-function toEsmName(name) {
-  return name.replace(/\.js$/, '.mjs');
-}
-
-function maybeToEsmName(name) {
-  return argv.esm ? toEsmName(name) : name;
+function getMinifiedName(name) {
+  if (argv.sxg) {
+    // For certain files during compilation, there seems to be files that get the extension written twice,
+    // e.g. sxg.sxg.js. This is the quickest fix for now.
+    return name.includes('.sxg.js') ? name : name.replace(/\.js$/, '.sxg.js');
```

Comment:
would `endsWith` be better here than includes?

Reply:
It would be more readable. I'll change this accordingly.